packages/terra-menu/src/Menu.jsx

```diff
@@ -53,12 +57,21 @@ const propTypes = {
    * Otherwise, the menu will display without an arrow and right aligned.
    */
   isArrowDisplayed: PropTypes.bool,
+  /**
+   * Allows assigning of root element custom data attribute for easy selecting of document base component.
```

Comment:
What does this mean for a consumer?

Reply:
Used [abstract modal ](https://github.com/cerner/terra-framework/blob/bed6ba818a71d660ae7416724fbd8a3293ecc0c6/packages/terra-abstract-modal/src/AbstractModal.jsx#L54)as reference

Reply:
I don't understand 'for easy selecting of document base component.' means for a consumer, tbh. When should the consumer set this and what should it be set to if not the root element?

Reply:
Changed the description in 15a749c